src/main/java/mate/academy/springboot/swagger/service/impl/ProductServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.springboot.swagger.service.impl;
+
+import java.math.BigDecimal;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.springboot.swagger.model.Product;
+import mate.academy.springboot.swagger.repository.ProductRepository;
+import mate.academy.springboot.swagger.service.ProductService;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class ProductServiceImpl implements ProductService {
+    private final ProductRepository productRepository;
+
+    @Override
+    public Product save(Product product) {
+        return productRepository.save(product);
+    }
+
+    @Override
+    public Product get(Long id) {
+        return productRepository.getById(id);
```

Comment:
Don't use `getById`, cause it's deprecated. Use `findById` instead and check if a value exists